build.sh

```diff
@@ -13,7 +13,7 @@ PATH="$TIZEN_STUDIO/tools/ide/bin:$PATH"
 #
 # Parse arguments
 #
-if [$8 -eq 'partner']; then
+if [ "$8" = "partner" ]; then
     PRIVILEGE=parner
```

Comment:
why is there no "t" in this one?

Reply:
Tizen has a bug, so we present the proper spelling as an input to our action then pass the wrongly spelled, but right value onward to Tizen.